docs/pages/cli/test.mdx

```diff
@@ -1,3 +1,29 @@
 # mud test
 
-**Coming soon**
+This command runs the tests in a MUD project. This requires several steps:
+
+1. Start an [`anvil`](https://book.getfoundry.sh/reference/anvil/) instance.
+   {/* This could be either an empty blockchain, or one that can be forked for a [fork test](https://book.getfoundry.sh/forge/fork-testing). */}
+1. Deploy the `World` and all related `System`s using [`mud deploy`](./deploy).
+1. Run tests using [`forge test`](https://book.getfoundry.sh/forge/tests).
```

Comment:
This sounds like these are the steps required by a user, but what we mean is "this is what the command does internally" right? Let's change the phrasing slightly to make that more clear

```suggestion
This command runs the tests in a MUD project. Internally, it runs the following steps:

1. Starts an [`anvil`](https://book.getfoundry.sh/reference/anvil/) instance.
   {/* This could be either an empty blockchain, or one that can be forked for a [fork test](https://book.getfoundry.sh/forge/fork-testing). */}
1. Deploys the `World` and all related `System`s using [`mud deploy`](./deploy).
1. Runs tests using [`forge test`](https://book.getfoundry.sh/forge/tests) and passes the deployed world address to the tests via the `WORLD_ADDRESS` environment variable.
```

I think we should also add a code snippet showing the usage of the `MudTest` base test contract (which loads the world address from the environment variable and provides it as variable in the test)